schema/workflow.yaml

```diff
@@ -664,7 +664,8 @@ $defs:
         basic:
           type: object
           oneOf:
-            - properties:
+            - title: BasicAuthInfo
```

Comment:
Wouldn't it be better to avoid abbreviations, especially for a descriptive property?